### PR TITLE
Test/workspace api

### PR DESCRIPTION
**Pull request by jsburckhardt**
Validation:

```bash
cd tests/acceptance/azure
export DATABRICKS_HOST="https://****.azuredatabricks.net"
export DATABRICKS_TOKEN="***"
go test -run "TestAzureWorkspace" .   
```

### CODE REVIEW

File: tests/acceptance/azure/config_test.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package azure_test
+
+import (
+	"os"
+
+	databricks "github.com/polar-rams/databricks-sdk-golang"
+	dbAzure "github.com/polar-rams/databricks-sdk-golang/azure"
+)
+
+var testConfig map[string]string
```
Comment (maniSbindra) [inline tests/acceptance/azure/config_test.go:10]: Can we remove testConfig if it is not needed anymore

**Comment by nedruk** [comment]
lgtm

File: tests/acceptance/azure/config_test.go [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+var c *dbAzure.DBClient
+
+const (
+	DATABRICKS_HOST_KEY  = "DATABRICKS_HOST"
+	DATABRICKS_TOKEN_KEY = "DATABRICKS_TOKEN"
+)
+
+func init() {
+	testConfig = make(map[string]string)
```
Comment (maniSbindra) [inline tests/acceptance/azure/config_test.go:19]: Can we remove test config from here as well

**Review by maniSbindra — CHANGES_REQUESTED** [review]
Minor change

**Pull request merged**